### ISSUE TITLE
WCAG 4.1.2: Buttons must have discernible text (.btn-default[type="submit"][title="Search"])

### ISSUE THREAD

**Issue by hash-logic** [hash-logic/issue#1]
**Issue**: `Buttons must have discernible text` ([`button-name`](https://dequeuniversity.com/rules/axe/3.2/button-name?application=msftAI))

**Target application**: [Home · Custom Portal](https://mymatterssit.microsoftcrmportals.com/)

**Element path**: #AllRFQ > .col-sm-2:nth-child(6) > form > .input-group > .input-group-btn > .btn-default[type="submit"][title="Search"]

**Snippet**:

    <button type="submit" class="btn btn-default" title="Search"> <i class="fa fa-search"> &nbsp; </i> </button>

**How to fix**:

    Fix all of the following:
      Element is in tab order and does not have accessible text

**Environment**:
Chrome version 73.0.3683.86

====

This accessibility issue was found using Accessibility Insights for Web 2.1.2 (axe-core 3.2.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
